data/json/items/gun/9mm.json

```diff
@@ -179,7 +179,7 @@
     "valid_mod_locations": [ [ "accessories", 4 ], [ "barrel", 1 ], [ "bore", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "muzzle", 1 ], [ "rail", 1 ], [ "sights", 1 ] ],
     "magazine_well": 1,
     "magazines": [
-      [ "9mm", [ "glockmag", "glockbigmag" ] ],
+      [ "9mm", [ "glock17_17", "glock17_22", "glock17_33", "glock_drum_50rd", "glock_drum_100rd" ] ], 
```

Comment:
So, what happens to old savegames that have a mag with the glockmag id in their gun? Or any references to these old ID's? Not sure if you can just rename these things

Reply:
Anyway, it seems the Kel-Tec SUB-2000 accepts any glock type magazine from 10 rounds an up (going from wikipedia) https://en.wikipedia.org/wiki/Kel-Tec_SUB-2000 so think you should just add the glockmag, and glockbigmag to the list as well.

Reply:
I copied the mag list from the glock-17. Whoops!